src/zap/zapper.cpp

```diff
@@ -1188,6 +1188,13 @@ void Zapper::InitializeCompilerFlags(CORCOMPILE_VERSION_INFO * pVersionInfo)
 
 #endif // _TARGET_X86_
 
+    // If we're compiling CoreLib, allow RyuJIT to generate SIMD code so that we can expand some
+    // of the hardware intrinsics.
+    if (m_pEECompileInfo->GetAssemblyModule(m_hAssembly) == m_pEECompileInfo->GetLoaderModuleForMscorlib())
+    {
+        m_pOpt->m_compilerFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_FEATURE_SIMD);
```

Comment:
This flag is only defined for targets that support SIMD (`_TARGET_X86_`, `_TARGET_AMD64_` and `_TARGET_ARM64_`). While we don't have any uses of ARM64 intrinsics currently in SPC.dll, I'd probably include that in this.

Reply:
It looks like that's causing the Arm failure.

Reply:
Yes it was. Thanks!